server/data/content_types.json

```diff
@@ -16,21 +16,20 @@
             "priority": {"required": true},
             "urgency": {"required": true},
             "anpa_category": {"required": true},
-            "category": {"required": true},
-            "subjects": {"required": true},
-            "genre_service" : {"required": true},
-            "places": {"required": true},
+            "subject": {"required": true},
+            "genre" : {"required": true},
+            "place": {"required": true},
             "ednote": {}
         },
         "editor": {
             "slugline": {"order": 1, "sdWidth": "half"},
             "priority": {"order": 2, "sdWidth": "quarter", "default": 6},
             "urgency": {"order": 3, "sdWidth": "quarter", "default": 5},
             "anpa_category": {"order": 4, "sdWidth": "full"},
-            "category": {"order": 5, "sdWidth": "half"},
-            "genre_service": {"order": 6, "sdWidth": "half"},
-            "subjects": {"order": 7, "sdWidth": "full"},
-            "places": {"order": 8, "sdWidth": "full"},
+            "category": {"order": 5, "sdWidth": "half", "schemafield": "subject", "required": true},
+            "genre_custom": {"order": 6, "sdWidth": "half", "schemafield": "genre", "required": true},
```

Comment:
maybe we can add this `schemafield` to controlled vocabulary schema, seems like this won't change across the app, will it?